src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+2.3.29
+------
+
+ * added PHP7-compatible versions for the constraints Null, True and False (IsNull, IsTrue, IsFalse)
+ * added PHP7-compatible versions for the constraint validators NullValidator, TrueValidator and FalseValidator (IsNullValidator, IsTrueValidator, IsFalseValidator)
```

Comment:
I propose a single line:
- fixed compatibility with PHP7 and up by introducing new constraints (IsNull, IsTrue, IsFalse) and related validators (IsNullValidator, IsTrueValidator, IsFalseValidator)